packages/stk/stk_balance/test_utils/MeshFixtureMxNRebalance.hpp

```diff
@@ -79,7 +79,7 @@ class MeshFixtureMxNRebalance : public stk::unit_test_util::MeshFixture
     void create_field_on_entire_mesh(const std::string& fieldName)
     {
         targetDecompField = &get_meta().declare_field<stk::mesh::Field<double> >(stk::topology::ELEMENT_RANK, fieldName, 1);
-        stk::mesh::put_field_on_mesh(*targetDecompField, get_meta().universal_part(), nullptr);
+        stk::mesh::put_field_on_mesh(*targetDecompField, get_meta().universal_part(), (double*)nullptr);
```

Comment:
It looks like `put_field_on_mesh` is templated on the type of the third argument, so `nullptr` won't suffice.  Still, I would prefer `static_cast<double*>` to the C-style cast.